src/ray/rpc/worker/core_worker_client.h

```diff
@@ -217,12 +211,27 @@ class CoreWorkerClient : public std::enable_shared_from_this<CoreWorkerClient>,
       : addr_(address) {
     grpc_client_ = std::make_unique<GrpcClient<CoreWorkerService>>(
         addr_.ip_address(), addr_.port(), client_call_manager);
+
+    retryable_grpc_client_ = RetryableGrpcClient::Create(
+        grpc_client_->Channel(),
+        client_call_manager.GetMainService(),
+        /*max_pending_requests_bytes=*/
+        std::numeric_limits<uint64_t>::max(),
+        /*check_channel_status_interval_milliseconds=*/
+        ::RayConfig::instance()
+            .gcs_client_check_connection_status_interval_milliseconds(),
```

Comment:
**`gcs_client`**`_check_connection_status_interval_milliseconds` hmm ok for a while, but longer term we can change to grpc_client_check_connection_status_interval_milliseconds